src/ray/util/test_util.h

```diff
@@ -58,6 +59,34 @@ std::shared_ptr<RayObject> GenerateRandomObject() {
   return std::shared_ptr<RayObject>(new RayObject(GenerateRandomBuffer(), nullptr));
 }
 
+/// Path to redis server executable binary.
+std::string REDIS_SERVER_EXEC_PATH;
+/// Path to redis client executable binary.
+std::string REDIS_CLIENT_EXEC_PATH;
+/// Path to redis module library.
+std::string REDIS_MODULE_LIBRARY_PATH;
+
+/// Test helper class, it will start redis server before the test runs,
+/// and stop redis server after the test is completed.
+class ManageRedisServiceForTest : public ::testing::Test {
```

Comment:
can you also change `GcsServerTest` to use this base class?

Reply:
Of course, that's part of the reason why this PR is still WIP.